test-module/src/test/scala/com/ing/baker/AllTypeRecipe.scala

```diff
@@ -0,0 +1,148 @@
+package com.ing.baker
+
+import com.ing.baker.recipe.common.{FiresOneOfEvents, ProvidesIngredient, ProvidesNothing}
+import com.ing.baker.recipe.scaladsl._
+
+/** This recipe is meant for testing purposes.
+  *
+i  * If you need a recipe that makes more sense then see the example package
+  *
+  * @see com.ing.baker.Examples
+  */
+
+object AllTypeRecipe {
+
+  case class User(name: String)
+
+  case class Payload(data: Map[String, String])
+  // @todo adding Map[String, Int] breaks the code
+//                     userMap: Map[String, Int])
```

Comment:
Enabling `Map[String, Int]` breaks the code. The `Int` in the value is not properly handled. This should be fixed or more explicitly be declined. (Guess just fixed).